src/test/scala/advancepassengerinfo/importer/processor/DqFileProcessorImplTest.scala

```diff
@@ -2,26 +2,29 @@ package advancepassengerinfo.importer.processor
 
 import advancepassengerinfo.generator.ManifestGenerator
```

Comment:
A few unused imports to remove

Reply:
I see this is used in line 73

Reply:
I didn't mean that one :) There are about 3 unused imports according to intellij. You can press Ctrl-Alt-O (or the mac equivalent) and intellij will remove unused ones for you